services/ingestion/app/ingest_match.py

```
@@ -0,0 +1,31 @@
+    logger.info("Start ingest match_id=%s", match_id)
+
+    client = provider or OpenDotaClient()
+
+    try:
+        raw = client.get_match(match_id)
+        contract = adapt_match(raw)
+        domain_match = parse_match(contract)
+        persist_match(domain_match)
+
+        logger.info("Ingest success match_id=%s", match_id)
+        return domain_match
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)